### PR TITLE
feat(tools): help button tests

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

Add tests that validate the functionality of the `Get Help` dropdown. Related to the necessary revert of React 17.

### CODE REVIEW

**Review by raisedadead — DISMISSED** [review]

**Comment by nhcarrigan** [comment]
I'm honestly shocked it worked with the missing `]` in the selectors.

**Review by raisedadead — DISMISSED** [review]

File: cypress/integration/learn/common-components/helpButton.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  it('should be visible', () => {
+    cy.visit(
+      '/learn/responsive-web-design/basic-html-and-html5/say-hello-to-html-elements'
+    );
+    cy.get('#get-help-dropdown').scrollIntoView().should('be.visible');
+  });
+
+  it('should toggle the dropdown menu', () => {
+    cy.visit(
```
Comment (ShaunSHamilton) [inline cypress/integration/learn/common-components/helpButton.js:16]: Does this need to be visited for each `it`? Or, are you just doing it as a way to unclick elements?

Comment (nhcarrigan) [reply cypress/integration/learn/common-components/helpButton.js]: I cleaned up the logic 🙂

**Review by raisedadead — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Pull request merged**